public/locales/en-US/teacher.json

```diff
@@ -1,3 +1,7 @@
 {
-  "noSubjectGroup": "No subjects"
-}
+  "title": "Teachers List",
+  "meetSoon": "Will meet soon",
+  "advisor": "Advisor of {{className}}",
+  "contacts": "Contacts",
+  "subjects": "Subjects"
+}
```

Comment:
You removed the key `noSubjectGroup`. Have you checked if this key is used anywhere?